tests/unit/dataactvalidator/test_scripts_loadTas.py

```diff
@@ -56,12 +59,14 @@ def test_cleanTas_space_nulls(tmpdir):
 
 
 def test_updateTASLookups(database, monkeypatch):
+    """Verify that TAS with the same account_num can be modified, that we
+    "close" any non-present TASes, and that we add new entries"""
     sess = database.session
     monkeypatch.setattr(
         loadTas, 'cleanTas', Mock(return_value=pd.DataFrame({
             'availability_type_code': ['0', '1', '2'],
-            'tas_id': [111, 222, 333],
-            'agency_identifier': ['0', '1', '2'],
+            'account_num': [111, 222, 333],
+            'agency_identifier': ['new', '1', 'new'],
```

Comment:
The no-doubt-unclear idea was to emphasize that these elements weren't in the original dataset. Let me rework the order of the calls here a little to try to make the logic clearer.